frontend/src/components/panels/project_filters_panel/filter_map_components/ProximitySlider.js

```diff
@@ -1,8 +1,26 @@
 import React from "react";
+import ReactDOM from 'react-dom';
+import Slider, { Range } from 'rc-slider';
+import 'rc-slider/assets/index.css';
+
+const marks = {
+	1: '1 mi',
+	5: '5 mi',
+	10: '10 mi',
+	20: '20 mi',
+};
+
+const dotStyle = {
+	'bottom': '-10px',
+    'width': '1px',
+    'border': '1px solid #e9e9e9',
+    'border-radius': '0%',
+    'margin-left': '-1px',
+};
 
 const ProximitySlider = () => (
     <div className="my-1">
-        <input type="range" className="custom-range" id="customRange1"/>
+        <Slider min={1} max={20} step={1} marks={marks} dotStyle={dotStyle} defaultValue={10} />
```

Comment:
The defaultValue, min and max should be dynamic state props for the this component as well. The API should help determine these values in the near future.